ecommerce-ios.xcodeproj/project.pbxproj

```diff
@@ -59,6 +62,9 @@
 
 /* Begin PBXFileReference section */
 		0578CFAAA271B29A3AEF6C65 /* Pods-ecommerce-ios.debug.xcconfig */ = {isa = PBXFileReference; includeInIndex = 1; lastKnownFileType = text.xcconfig; name = "Pods-ecommerce-ios.debug.xcconfig"; path = "Target Support Files/Pods-ecommerce-ios/Pods-ecommerce-ios.debug.xcconfig"; sourceTree = "<group>"; };
+		222EB87D26525D2B00BF46BD /* SearchBarView.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = SearchBarView.swift; sourceTree = "<group>"; };
+		222EB885265268BA00BF46BD /* View+CustomOverlayView.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = "View+CustomOverlayView.swift"; sourceTree = "<group>"; };
+		222EB88D2652723B00BF46BD /* String+Ultility.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = "String+Ultility.swift"; sourceTree = "<group>"; };
```

Comment:
String+Utility ?

Reply:
It's like a helper actually. 

![image](https://user-images.githubusercontent.com/23162627/118611741-5b2df900-b7e7-11eb-9032-d14d471222f1.png)

Reply:
I think the class name is misspelled